misc/config_tools/schema/checks/ivsh_memory.xsd

```diff
@@ -10,7 +10,7 @@
                   every $REGION_NAME in /acrn-config//IVSHMEM_REGION[IVSHMEM_VMS/IVSHMEM_VM/VM_NAME=$VM_NAME]/NAME satisfies
                   count(/acrn-config//IVSHMEM_REGION[NAME=$REGION_NAME]/IVSHMEM_VMS/IVSHMEM_VM[VM_NAME=$VM_NAME]) = 1">
     <xs:annotation acrn:severity="error" acrn:report-on="$REGION_NAME">
-      <xs:documentation>InterVM shared memory region name "{$REGION_NAME}" should be different for the same VM "{$VM_NAME}"</xs:documentation>
+        <xs:documentation>VM "{$VM_NAME}" may not specify InterVM shared memory region name "{$REGION_NAME}" more than once.</xs:documentation>
```

Comment:
The original description is quite confusing. What this assertions actually means is that "VM {$VM_NAME} cannot be duplicated in the list of VMs that have access to the inter-VM shared memory region {$REGION_NAME}"

Reply:
got it.  thanks